docker-compose.yml

```diff
@@ -1,7 +1,8 @@
 version: '3'
 services:
     mci:
-        image: brighthive/master-client-index:1.0.5
+        # image: brighthive/master-client-index:1.0.5
+        image: brighthive/master-client-index:1.0.6-test
```

Comment:
Will you update this (i.e., remove `-test`) when you cut a new version?

Reply:
Absolutely. That sort of ended up there as an artifact of needing to deploy it to the sandbox for Virginia work in order to unblock Kalika.